buildbot/master/files/config/factories.py

```diff
@@ -58,31 +58,38 @@ class DynamicServoFactory(ServoFactory):
 
     def __init__(self, builder_name, environment):
         self.environment = environment
+        self.is_windows = True if re.match('windows(.*)',
```

Comment:
We can do `self.is_windows = re.match(...) is not None` directly instead. Also, no need for the parentheses around the `.*`.